tests/scan.rs

```diff
@@ -71,3 +74,208 @@ async fn test_simple_scan() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_consistent_scan() -> Result<(), Box<dyn std::error::Error>> {
+    let mut tm = util::setup().await?;
+    let table_name = tm
+        .create_temporary_table_with_items("pk,S", None, [TemporaryItem::new("1", None, None)])
+        .await?;
+
+    let mut c = tm.command()?;
+    let scan_cmd = c.args(["--region", "local", "--table", &table_name, "scan"]);
+    let scan_exec = scan_cmd
+        .assert()
+        .success()
+        .stdout(predicate::str::contains("1"));
+
+    let scan_consistent_cmd = scan_cmd.args(["--consistent-read"]);
+    scan_consistent_cmd
+        .assert()
+        .success()
+        .stdout(scan_exec.get_output().stdout.to_owned());
+
+    Ok(())
+}
+
+#[tokio::test]
+async fn test_index_scan() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
The test is duplicated with `test_scan_with_index`.

Reply:
Thank you for pointing it out. I have fixed.